code/game/data_huds.dm

```diff
@@ -17,7 +17,7 @@
 
 /datum/atom_hud/data
 
-/datum/atom_hud/data/client
+/datum/atom_hud/data/human/client
```

Comment:
Why this repathing?

Reply:
everything in /datum/atom_hud//human/ will be auto  assigned to humans while spawning that's why i did the repath